day_6/exercises/burrito.js

```diff
@@ -16,12 +16,26 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+  addTopping() {
```

Comment:
Have you considered passing the new topping as an argument / parameter to this class method `addTopping`?  For example, you could do the following:
```
addTopping(newTopping) {
  this.toppings.push(newTopping);
}
```
And then when you call the method you could pass that argument in as you did on line 33.